packages/strapi-design-system/src/Checkbox/Checkbox.js

```diff
@@ -12,7 +12,7 @@ const CheckboxLabel = styled(Text)`
   display: flex;
   align-items: flex-start;
   * {
-    cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'default')};
+    cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
```

Comment:
About that update, the default cursor on checkboxes is the default behaviour of html checkboxes. IMO, if we do that modification here, we should standardize this modification also in other clickable components (Toggle, etc) @maevalienard

Reply:
Apart from Checkbox and toggle, what else is impacted? :)

Reply:
Checkbox, Toggle, Radio button (maybe the Select list item too)

Reply:
That'd make sense to me to have a cursor change on all these :) @HichamELBSI

Reply:
I created a task for the other components on the migration feedback board. @ronronscelestes